### PR TITLE
changes to mobile KPI views

### PR DESCRIPTION
**Pull request by irrationalagent**
The forecasting code for the mobile KPIs wasn't quite right. This fixes it.

I also have a working version of the mobile KPI dashboard that depends on these changes. Once This is merged, I'll throw that on a shared branch so we can hack on it together.

### CODE REVIEW

File: KPI/views/mobile_usage_forecast.view.lkml [hunk 1]
```diff
@@ -4,15 +4,14 @@ view: mobile_dau_model {
...
-      {% condition mobile_usage_2021.canonical_app_name %} canonical_app_name {% endcondition %}
       {% condition mobile_usage_2021.channel %} channel {% endcondition %}
       {% condition mobile_usage_2021.country %} country {% endcondition %}
       {% condition mobile_usage_2021.country_name %} country_name {% endcondition %}
       {% condition mobile_usage_2021.distribution_id %} distribution_id {% endcondition %}
       {% condition mobile_usage_2021.id_bucket %} id_bucket {% endcondition %}
       {% condition mobile_usage_2021.os %} os {% endcondition %}
       -- add source or campaign not null as attributed
-      -- country tier: FR, GB, US, CA, DE
+      -- country tier: FR, GB, US, CA, DE, CA
```
Comment (fbertsch) [inline KPI/views/mobile_usage_forecast.view.lkml:14]:
```suggestion
      -- country tier: FR, GB, US, CA, DE
```
CA is already in the list here.

**Review by fbertsch — COMMENTED** [review]
I think I'm missing some context here w.r.t. the associated dashboard

File: KPI/views/mobile_usage_forecast.view.lkml [hunk 2]
```diff
@@ -162,10 +169,22 @@ view: mobile_prediction {
     sql: ANY_VALUE(${TABLE}.dau_forecast) ;;
   }
 
+  measure: dau_forecast_7day_ma {
+    type: number
+    value_format: "#,##0"
+    sql: ANY_VALUE(${TABLE}.dau_forecast_7day_ma) ;;
```
[the remark this answers is not in the document]
Comment (irrationalagent) [reply KPI/views/mobile_usage_forecast.view.lkml:175]: the moving averages are meant to be used in the KPI dashboard - to be consistent with desktop (I changed the daily charts on the desktop dash to be moving averages while you were out due to a request).

File: KPI/views/mobile_usage_forecast.view.lkml [hunk 3]
```diff
@@ -162,10 +169,22 @@ view: mobile_prediction {
     sql: ANY_VALUE(${TABLE}.dau_forecast) ;;
   }
 
+  measure: dau_forecast_7day_ma {
```
[the remark this answers is not in the document]
Comment (irrationalagent) [reply KPI/views/mobile_usage_forecast.view.lkml:172]: Do you think we should hide the moving averages or the raw values? The MAs are what are intended to be shown in the dash, but people might want to know the exact values too.

File: KPI/views/mobile_usage_forecast.view.lkml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fbertsch) [reply KPI/views/mobile_usage_forecast.view.lkml:175]: Leif and I agreed these should be kept as they are, but hidden.

File: KPI/views/mobile_usage_forecast.view.lkml (hunk 3 above)
[the remark this answers is not in the document]
Comment (fbertsch) [reply KPI/views/mobile_usage_forecast.view.lkml:172]: I think we should hide the moving average. The raw values are what may be interesting, and people can use [table calcs](https://docs.looker.com/exploring-data/using-table-calculations) to derive moving average.

**Review by fbertsch — APPROVED** [review]
r+wc. Other things we discussed in slack:
- Descriptions for all fields
- Hiding all non-user-facing fields (i.e. ones that are just for this dashboard, and not for public consumption)
- A dash element to show whether the currently selected dimensions are building a model

**Pull request merged**